src/main/java/uk/gov/ons/fwmt/census/rmadapter/canonical/CanonicalJobHelper.java

```diff
@@ -38,57 +34,54 @@ public FWMTCreateJobRequest createJob(ActionInstruction actionInstruction) throw
     // TODO not yet implemented in Canonical
     //address.setCategory(actionAddress.getCategory());
 
-    fwmtCreateJobRequest.setJobIdentity(actionRequest.getCaseRef()); 
-    fwmtCreateJobRequest.setSurveyType(actionRequest.getSurveyRef()); 
+    createJobRequest.setJobIdentity(actionRequest.getCaseRef()); 
```

Comment:
this should be caseId and case ref needs added to create job request. Both are always required so neither should be an additional property

Reply:
CaseRef is mapped to caseRef and CaseId is an additional property